docs/users/navigation/joysticks.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Joystick Navigation
+layout: default
+
+grand_parent: Users
+parent: Navigation
+nav_order: 2
+---
+
+# Joystick Navigation
+In addition to the normal navigation in OpenSpace using the keyboard and mouse it is also possible to navigate using a game controller, such as an Xbox controller, PS4 controller or a SpaceMouse. If you want to use any of these supported controllers you need to include their asset file into a profile. This can be done in the profile editor in the launcher by editing the profile to include one of the joystick assets. All of the joystick assets are located in the sub-folder <code>data/assets/util/joysticks</code> inside the OpenSpace folder. It is important that the right asset file is included for the type of game controller you are using. For example the wireless Xbox asset will not work with a non-wireless Xbox controller and vice versa.
+
+## Xbox Controller
+The navigation using an Xbox or PS4 controller are very similar, the only difference is the layout of the controllers. Otherwise the functionality is the same. The image below shows a map over the buttons and joysticks on a Xbox controller.
```

Comment:
This comes a bit out of the blue. Put Xbox after PS¤ ? Then the "röd tråd" is preserved and the first sentence here makes more sense

Reply:
I don't really under stand this comment. What part of the sentence are you referring to?

Reply:
Since the Xbox controller is the most commonly used, I would like that to still be before the PS4 controller. But I agree and will move the comparison part down to the PS4 section instead.